.github/workflows/meson.yml

```diff
@@ -68,15 +69,27 @@ jobs:
         uses: ilammy/msvc-dev-cmd@v1
 
       - name: Install dependecies with choco
-        if: matrix.config.pkgmanager == 'choco'
+        if: matrix.config.shell == 'powershell'
         run: |
           # for whatever reason choco insist on a reboot if it installs meson
-          choco install ninja nasm python
+          choco install ninja nasm python pkgconfiglite
           python -m pip install --upgrade pip
           pip install meson
 
+      - name: Install dependecies with msys
+        if: matrix.config.shell == 'powershell'
```

Comment:
I’m not sure about this and other `== 'powershell'` checks. At the one hand, all current `powershell` checks actually target “compiling with MSVC” and checking the latter would be more idiomatic. On the other hand, _if_ for whatever reason we ever add a “gcc build in powershell” runner it would probably need the same or a very similar treatment.

Another nit, to avoid confusion with native msys2 package installation “Install msys dependencies for PowerShell” might be a better step name